projects/material-addons/src/lib/button/flat-button/link-button.component.cy.ts

```diff
@@ -0,0 +1,56 @@
+import { ButtonModule } from '../button.module';
+import { MatButtonModule } from '@angular/material/button';
+import { Component } from '@angular/core';
+
+const checkButtonBasicAttributes = () => {
```

Comment:
I think this is almost a duplicate from above? Maybe we can have only one of them taking some arguments to be configured?

Reply:
fixed with button.component.cy.ts tests